.github/workflows/test.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
     strategy:
       matrix:
         rust:
+          - "1.64.0"
```

Comment:
I think that "1.64" to set semver compatible toolchains is better.  It looks like 1.64 happens to have only had one release, but if lalrpop later updates to testing 1.66, we'd want to get 1.66.1 instead of 1.66.0, and just specifying major.minor should get us the latest point release.

Reply:
Thank you!